doc/connectivity/networking/conn_mgr/main.rst

```diff
@@ -73,6 +73,17 @@ Afterwards, ifaces can become ready or unready without firing additional events,
 
 When there are no longer any ready ifaces left, the :c:macro:`NET_EVENT_L4_DISCONNECTED` :ref:`network management <net_mgmt_interface>` event is triggered, and IP connectivity is said to be unready.
 
+.. note::
+
+   Connection Manager also fires the following more specific ``CONNECTED`` / ``DISCONNECTED`` events:
+
+   - :c:macro:`NET_EVENT_L4_IPV4_CONNECTED`
+   - :c:macro:`NET_EVENT_L4_IPV4_DISCONNECTED`
+   - :c:macro:`NET_EVENT_L4_IPV6_CONNECTED`
+   - :c:macro:`NET_EVENT_L4_IPV6_DISCONNECTED`
```

Comment:
```suggestion
   - :c:enumerator:`NET_EVENT_L4_IPV4_CONNECTED`
   - :c:enumerator:`NET_EVENT_L4_IPV4_DISCONNECTED`
   - :c:enumerator:`NET_EVENT_L4_IPV6_CONNECTED`
   - :c:enumerator:`NET_EVENT_L4_IPV6_DISCONNECTED`
```

Reply:
Are you certain these should be changed to `:c:enumerator:`? They are defined as macros whose values derive from the actual enumerator values

Reply:
Of course! Read too fast and missed the difference between the "CMD" enumerators ans the macros